dhis-2/dhis-support/dhis-support-hibernate/src/main/java/org/hisp/dhis/datasource/DatabasePoolUtils.java

```diff
@@ -176,13 +178,16 @@ public ConfigKeyMapper getMapper() {
     }
   }
 
+  private DatabasePoolUtils() {}
+
   public static DataSource createDbPool(PoolConfig config)
       throws PropertyVetoException, SQLException {
     Objects.requireNonNull(config);
 
     DbPoolType dbType = DbPoolType.valueOf(config.dbPoolType.toUpperCase());
+    log.info(String.format("Database pool type value is [%s]", dbType));
```

Comment:
no need for the string format, logging can handle the param e.g.
`log.info("Database pool type value is {}", dbType);`